### ISSUE TITLE
Add version and/or date_added fields to items

### ISSUE THREAD

**Issue by dmitrizagidulin** [MarceloAlves/StarboundItems#28]
This would especially makes sense once the process for adding new items (from resources, etc) gets automated. But even before then --

Add optional 'version' field (for Starbound version) and 'date_added' field (to denote when an item was added to the nightly build) to the items list.

**Comment by MarceloAlves** [comment]
This is doable but I stopped updating from nightly a while back ago. Since it takes a good 45 minutes or so to update I just update when Stable releases happen. If the process can be fully automated then updating nightlies wouldn't be a problem.

What about items that are already there? Would their version/date just be the first time the list is generated?

**Comment by dmitrizagidulin** [comment]
Yeah, hmm, the back-dating existing items would require a bit of manual work. Unless we can figure out how to reliably parse the wiki entries (specifically, the version history section). Will ponder..

**Comment by MarceloAlves** [comment]
MediaWiki has a built in export that will give you a XML of the page. Adding `Special:Export` in the URL gets you the data. Example: http://starbounder.org/Special:Export/Coal

But not all entries have the version listed of when they were added. I can try reaching out and seeing if they'd be willing to export that data instead of slamming the site for 4k items.

**Comment by dmitrizagidulin** [comment]
Ah, yeah, an export would be brilliant. (Like they export the object tags (I assume you're using that feed to populate the tags field?))

**Comment by MarceloAlves** [comment]
The tags are included in the Starbound assets
